docs/contributing/index.md

```diff
@@ -92,3 +92,50 @@ Once your changes are ready, click on the "Commit changes..." button in the top
 
     * if the change is small and you are CSCS staff, you can merge the PR immediately
     * all other changes can be
+
+## Style Guide
+
+This section contains general guidelines for how to format and present documentation in this repository.
+They should be followed for most cases, but as a guideline it can be broken, _with good reason_.
+
+### General Formatting
+
+- Format paragraphs with one sentence per line for easier diffs.
+- Leave a space before and after headings.
+
+### Headings are Written in Title Case
+
+Use [title case](https://en.wikipedia.org/wiki/Letter_case#Title_case) for headings, meaning all words are capitalized except for minor words.
+
+### Avoid Nesting Headings too Deep
+
+Nesting headings up to three levels is generally ok.
+
+### Unordered and Ordered Lists
+
+Write lists as proper sentences.
+Separate the items simply with commas if each item is simple, or make each item a full sentence if the items are longer and contain multiple sentences.
+
+1. The first item can look like this,
+2. The second like this, and
+3. The third item like this.
```

Comment:
I would put the `and` at the start of the third line, and star each line with a lower case letter (the whole list is a sentence, no?)

I agree that care should be taken with formating lists, but it is tricky to canonicalize without being too strict. For example, I would format your list as follows:

contain multiple sentences:

1. the first item can look like this;
2. the second like this;
3. and the third like this.

Reply:
> but it is tricky to canonicalize without being too strict
This is probably key, so in the end these are just guidelines.

Agree on capitalization (see https://github.com/eth-cscs/cscs-docs/pull/53#discussion_r2007514552). I feel like it's more common to have the "and" or "or" trailing on the second-to-last item though?

Reply:
Honestly, I don't know.
It feels wrong to me, because if we use the oxford comma (as we should), the and comes after the comma.

I think if people go to enough effort to punctuate their lists with care, that's enough?

Reply:
If it's all the same to you, I'd leave it with the trailing and/or on the second-to-last item, but indeed, let's not _actually_ bikeshed this too much.

> I think if people go to enough effort to punctuate their lists with care, that's enough?

Yes, of course.